sql/moz-fx-data-shared-prod/firefox_ios_derived/funnel_ios_v1/query.sql

```diff
@@ -0,0 +1,76 @@
+WITH views_data AS (
+  SELECT
+    `date`,
+    territory AS country_name,
+    SUM(impressions_unique_device) AS views,
+  FROM
+    app_store.firefox_app_store_territory_source_type_report
```

Comment:
I noticed these we previously using resources from `mozdata.analysis` was there anything special about these tables @bani ?